### ISSUE TITLE
Error: image wscherphof/oracle-12c not found

### ISSUE THREAD

**Issue by HonoluluHenk** [wscherphof/oracle-12c#24]
Hi

Thanks for your effort building this great image.
Unfortunately, I cannot pull the image anymore:

docker pull wscherphof/oracle-12c
just gives the following output:
Pulling repository docker.io/wscherphof/oracle-12c
Error: image wscherphof/oracle-12c not found

I cannot find this image on the docker hub, neither.

Just for reference: using docker 1.10.1 on latest Ubuntu

**Comment by wscherphof** [comment]
Yeah Oracle made Docker to remove the image. So you'll need to build it
yourself..
Op di 16 feb. 2016 om 19:09 schreef Christoph Linder <
notifications@github.com>

> Hi
>
> Thanks for your effort building this great image.
> Unfortunately, I cannot pull the image anymore:
>
> docker pull wscherphof/oracle-12c
> just gives the following output:
> Pulling repository docker.io/wscherphof/oracle-12c
> Error: image wscherphof/oracle-12c not found
>
> I cannot find this image on the docker hub, neither.
>
> Just for reference: using docker 1.10.1 on latest Ubuntu
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wscherphof/oracle-12c/issues/24>.
>
-- 
Met vriendelijke groet/Kind regards,
Wouter Scherphof
